themes/jb/layouts/calendar/single.html

```diff
@@ -320,24 +317,120 @@ <h1>{{.Title}}</h1>
           <option value="Pacific/Tongatapu">(GMT+13:00) Tongatapu</option>
           <option value="Pacific/Kiritimati">(GMT+14:00) Kiritimati</option>
         </select>
+
+        <br>
+        <br>
+
+        <div class="container">
+          <div class="columns is-multiline">
+             {{ range where .Site.Pages  "Section" "show" }}
+              {{ if (isset .Params "live") }}
+                <div class="column is-6 is-3-fullhd is-3-desktop is-12-mobile"  style="display: flex;">
+                  {{ partial "episode/schedule.html" . }}
+                </div>
+              {{ end }}
+            {{ end }}
+          </div>
+        </div>
+
+        <br>
+        <br>
+
+        Subscribe To The Calendar: <a href="{{.Site.Params.calendar.rss}}">iCalendar</a>
+
+        <iframe frameborder="0" scrolling="no" data-src="//www.google.com/calendar/embed?showTitle=0&amp;showNav=0&amp;showPrint=0&amp;showTabs=0&amp;showCalendars=0&amp;mode=AGENDA&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src={{.Site.Params.calendar.embedd}}&amp;color=%235229A3" width="100%" height="400px"></iframe>
+
+
     </div>
   </div>
 
   <script>
```

Comment:
Since the JavaScript as increased significantly, I think it should be moved to its own JS file and imported to the page. 

Here is how the jb live stream handles its JavaScript

```html
{{ define "headcontent" }}
  {{ $jbTube := resources.Get "js/jb-live.js" }}
  {{ if hugo.IsProduction }}
    {{ $jbTube = $jbTube | minify | fingerprint }}
  {{ end }}
<script type="text/javascript" src="{{ $jbTube.Permalink }}" integrity="{{ $jbTube.Data.Integrity }}"></script>
```